app/assets/javascripts/preferences/panes/two-factor-auth/TwoFactorActivation.ts

```diff
@@ -0,0 +1,127 @@
+import { action, makeAutoObservable, observable, untracked } from 'mobx';
+import { MfaGateway } from './MfaProps';
+
+const activationSteps = [
+  'scan-qr-code',
+  'save-secret-key',
+  'verification',
+] as const;
```

Comment:
Maybe we could store activation steps in an enum and use its values instead of the strings. In that case we could avoid defining new `ActivationStep` type at line 10 and just use that enum as _activationStep's type.

Reply:
Ah, actually we don't need the list of steps any more. Changed to string literal type.